torch/csrc/autograd/engine.cpp

```diff
@@ -216,7 +232,8 @@ auto ReadyQueue::pop() -> FunctionTask {
   return task;
 }
 
-Engine::Engine() = default;
+// This limit is based on the default python recursion limit which is 1000
```

Comment:
comments do not match the code 1000 vs 100

Reply:
We need to set it lower than the actual python limit to take into account the function calls within python